packages/web-styles/src/components/form-range.scss

```
@@ -43,3 +44,46 @@
     transform: translateY(#{-0.5 * forms.$form-range-height});
   }
 }
+
+@include utilities.high-contrast-mode() {
+  .form-range {
+    // linear-gradient is forced to the value of "none" in forces-color mode
+    // so, the "forced-color-adjust" property is necessary for "linear-gradient" to continue to work
+    forced-color-adjust: none;
+
+    &:not(:disabled):not(.disabled) {
```

```suggestion
    &:not(:disabled, .disabled) {
```
The :not pseudo selector accepts lists now 🎉 https://caniuse.com/css-not-sel-list

---

I just copied the existing selector ;)
So I will change both of them.